src/api/evaluators/custom_evals/eval_combined.py

```diff
@@ -0,0 +1,333 @@
+import json
+import os
+from datetime import timedelta
+from typing import Any, Dict, List
+from azure.core.exceptions import HttpResponseError
+from azure.identity import DefaultAzureCredential
+from azure.monitor.query import LogsQueryClient, LogsQueryStatus
+from dotenv import load_dotenv
+import prompty
+import prompty.azure
+import logging
+
+# Configure logging
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
+
+
+# Load environment variables
+load_dotenv()
+
+
+
+# Configuration for the Azure OpenAI endpoint
+model_config = {
```

Comment:
check these are defined or exit with a message that they are required

Reply:
fixed.